### PR TITLE
tf.nn.compute_average_loss (no global_batch_size)

### LINKED ISSUE #10524: compute_average_loss

### PR DESCRIPTION
**Pull request by tibsor**
Close #10524

### CODE REVIEW

**Comment by tibsor** [comment]
Hello,

In order to implement `global_batch_size` in the `tf.nn.compute_average_loss` function, a way to select the [training strategy](https://www.tensorflow.org/api_docs/python/tf/distribute/Strategy) should be implemented, as a part of TensorFlow's [distributed training API](https://www.tensorflow.org/guide/distributed_training). I have not been able to find any issues that address this.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.
